client/stylesheets/_toCat.scss

```diff
@@ -3272,6 +3272,19 @@ On click the images will be displayed at normal size to complete the effect
     height: 500px !important;
 }
 
+.itemAfterBanner {
+    background-color: $color-purple;
+    padding: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border-radius, margin-bottom, padding, width